instance/operation_runner_test.go

```diff
@@ -17,17 +17,38 @@
 package instance_test
 
 import (
-	"errors"
-
-	"code.cloudfoundry.org/cli/plugin/models"
+	"code.cloudfoundry.org/cli/plugin"
 	"code.cloudfoundry.org/cli/plugin/pluginfakes"
+	"errors"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/instance"
 	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient"
 	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient/httpclientfakes"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/instance"
 )
 
+type FakeOperation struct {
```

Comment:
Let's stay consistent and use counterfeiter to generate fakes rather than handcrafting them - see `httpclient/client.go` as an example

Reply:
OK :)